readme.md

```diff
@@ -46,6 +46,21 @@ erDiagram
         int avatar_file_id FK
     }
 
+    tasks {
```

Comment:
We already have `tasks` in README

Reply:
I added one because I  created a migration. Then Margarita added.
Now only one.